scipy/optimize/_differentialevolution.py

```diff
@@ -294,6 +295,53 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     iteration, rather than multiple times for all the population members; the
     interpreter overhead is reduced.
 
+    An outline of the strategies and their stengths and weaknesses is given by 
+    Qiang and Mitchell (2014) [8]_
+
+    ..math::
+            DE/rand/1 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3})
```

Comment:
In equation on line ~260 we use a different notation. Can you convert all the equations you've added to use the same notation?
The reason is that $F_{xc}$ is actually the `mutation` parameter, $F_{cr}$ is the `recombination` parameter, etc.
 I'd like to use terminology that is actually used in the parameters.